apps/frontend/src/app/Data/Characters/Nilou/index.tsx

```diff
@@ -198,8 +198,8 @@ export const dmgFormulas = {
     dance2Dmg: dmgNode('hp', dm.skill.dance2Dmg, 'skill'),
     whirl1Dmg: dmgNode('hp', dm.skill.whirl1Dmg, 'skill'),
     whirl2Dmg: dmgNode('hp', dm.skill.whirl2Dmg, 'skill'),
-    moonDmg: dmgNode('hp', dm.skill.moonDmg, 'skill', {
-      premod: { skill_dmg_: c1_moon_dmg_ },
+    illusionDmg: dmgNode('hp', dm.skill.illusionDmg, 'skill', {
```

Comment:
Breaks any multi opts. we can change the dm value, but maybe keep the first illusionDmg instance